File: java/app/src/main/java/net/fbvictorhugo/j/barreirasanitaria/data/dao/IQuestionarioDAO.java
package net.fbvictorhugo.j.barreirasanitaria.data.dao;

import net.fbvictorhugo.j.barreirasanitaria.data.model.Questionario;

import java.util.List;

public interface IQuestionarioDAO {

    List<Questionario> listar();

    void inserir(Questionario questionario) throws Exception;

}

File: kotlin/app/src/main/java/net/fbvictorhugo/k/barreirasanitaria/data/dao/IQuestionarioDAO.kt
package net.fbvictorhugo.k.barreirasanitaria.data.dao

import net.fbvictorhugo.k.barreirasanitaria.data.model.Questionario

interface IQuestionarioDAO {
    fun listar(): ArrayList<Questionario>

    @Throws(Exception::class)
    fun inserir(questionario: Questionario)

}